# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/webrtc-audio-processing-1/hotfix-c++17.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- webrtc-audio-processing-v2.1/meson.build.vanilla	2026-09-01 12:08:05.231188309 +0300
+++ webrtc-audio-processing-v2.1/meson.build	2026-09-01 12:08:10.816326658 +0300
@@ -4,7 +4,7 @@
   default_options : [ 'warning_level=1',
                       'buildtype=debugoptimized',
                       'c_std=c11',
-                      'cpp_std=c++17',
+                      'cpp_std=c++20',
                     ]
 )
 

